tests/chainermn_tests/communicator_tests/test_communicator.py

```diff
@@ -307,3 +308,24 @@ def test_alltoall_cpu(self):
     def test_alltoall_gpu(self):
         self.setup(True)
         self.check_alltoall()
+
+    def check_allreduce(self):
+        x = np.arange(18) + self.communicator.rank
+        xs = x.astype(np.float32)
+        xs = self.communicator.allreduce(xs)
+
+        sum = reduce(lambda a, b: a + b,
```

Comment:
Why do you use `reduce` instead of `sum`?

Reply:
I just didn't know that and updated.